nextage_moveit_config/test/test_moveit.py

```diff
@@ -290,7 +290,7 @@ def test_botharms_plan(self):
         target_pose_l.orientation.w = q[3]
         botharms.set_pose_target(target_pose_r, 'RARM_JOINT5_Link')
         botharms.set_pose_target(target_pose_l, 'LARM_JOINT5_Link')
-        self.assertTrue(botharms.go())
+        self.assertTrue(botharms.go() or botharms.go() or botharms.go())
```

Comment:
no, just for safety

--
◉ Kei Okada


On Sun, Sep 4, 2016 at 9:39 AM, Isaac I.Y. Saito <notifications@github.com>
wrote:

> In nextage_moveit_config/test/test_moveit.py
> <https://github.com/tork-a/rtmros_nextage/pull/252#discussion_r77444994>:
>
> > @@ -290,7 +290,7 @@ def test_botharms_plan(self):
> >          target_pose_l.orientation.w = q[3]
> >          botharms.set_pose_target(target_pose_r, 'RARM_JOINT5_Link')
> >          botharms.set_pose_target(target_pose_l, 'LARM_JOINT5_Link')
> > -        self.assertTrue(botharms.go())
> > +        self.assertTrue(botharms.go() or botharms.go() or botharms.go())
>
> Does this mean that the success rate of botharms.go() with this
> particular condition is 1/3?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/tork-a/rtmros_nextage/pull/252/files/36b50eddf1f5342977362f2748e5cf46b553c3a6#r77444994>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAeG3H0g_I0OP3S2jA73u0IVTrzR-q7xks5qmhNQgaJpZM4J0U1->
> .
>